.backportrc.json

```diff
@@ -2,14 +2,17 @@
   "repoOwner": "microsoft",
   "repoName": "ccf",
   "targetBranchChoices": ["release/2.x"],
-
   "branchLabelMapping": {
     "^(.+)-todo$": "release/$1"
   },
   "autoMerge": true,
   "autoMergeMethod": "squash",
   "prTitle": "[{targetBranch}] Cherry pick: {commitMessages}",
-  "prDescription": "Backports the following commits to {targetBranch}:\n{commitMessages}",
+  "prDescription": "Backports the following commits to `{targetBranch}`:\n{commitMessages}",
   "sourcePRLabels": ["backported"],
-  "targetPRLabels": ["backport"]
+  "targetPRLabels": ["2.x-backport"],
```

Comment:
Hardcoded for now but I've asked to support dynamic target labels: https://github.com/sqren/backport/issues/391